tests/llama_index/test_llama_index_tracer.py

```diff
@@ -180,72 +256,43 @@ def test_trace_retriever(multi_index, is_async):
     assert spans[3].attributes["model_name"] == Settings.embed_model.model_name
 
 
-@pytest.mark.parametrize("is_async", [True, False])
-def test_trace_query_engine(multi_index, is_async):
-    engine = multi_index.as_query_engine()
-
-    response = asyncio.run(engine.aquery("Hello")) if is_async else engine.query("Hello")
-    assert response.response.startswith('[{"role": "system", "content": "You are an')
-    response = asdict(response)
+@pytest.mark.parametrize(
+    ("is_stream", "is_async"),
+    [
+        (False, False),
+        (True, False),
+        (False, True),
+        # (True, True),  # Async stream is not supported yet
+    ],
+)
+def test_trace_query_engine(multi_index, is_stream, is_async):
```

Comment:
```suggestion
@pytest.mark.parametrize("is_stream", [False, True])
@pytest.mark.parametrize("is_async", [False, True])
def test_trace_query_engine(multi_index, is_stream, is_async):
    if is_stream and is_async:
        pytest.skip("Async stream is not supported yet")
```

not a blocker, but I'd use nested parametrization here.